dist/jowl.spec

```diff
@@ -0,0 +1,87 @@
+%global npm_name jowl
+
+Name:           %{npm_name}
+Version:        2.0.2
+Release:        1%{?dist}
+Summary:        CLI for JSON operations with Lodash
+
+License:        MIT
+URL:            https://jowl.app
+Source0:        https://github.com/daxelrod/%{npm_name}/archive/refs/tags/v%{version}.tar.gz
```

Comment:
You should use the URL form documented by Fedora here: https://docs.fedoraproject.org/en-US/packaging-guidelines/SourceURL/#_git_tags